src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -468,6 +468,17 @@ internal static string GetKeyForDeletionInProgress(string node, int? shardNumber
 
             return $"{node}${shardNumber.Value}";
         }
+
+        public bool IsShardBeingDeletedOnAnyNode(int shardNumber)
```

Comment:
internal?

Reply:
done

Reply:
in https://github.com/ravendb/ravendb/pull/15577